HostAbroadTests/src/test/systemAcceptanceTests/features/SearchTraveler.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Buscar usuario (viajero).
+
+  Descripción: Como usuario registrado, no registrado o administrador quiero
```

Comment:
Pls change this with the new description from the PB.

Reply:
This cabr be de exact same because of the errors while executing the tests, but it is a copy from PB adding "viajero" at the end of 'Given' and 'And'